### PR TITLE
TAL-21: Create Withdraw Page test cases

### PR DESCRIPTION
**Pull request by Manuyafox**

### CODE REVIEW

File: src/test/resources/WithdrawPageTestCases [hunk 1]
```diff
@@ -0,0 +1,72 @@
+1. Withdraw transaction happy flow
+Pre-condition:
+- Navigate to Puppy Bank login page.
+- Login as admin.
+- Automatic transition to the dashboard page occurs.
```
Comment (iiddd) [inline src/test/resources/WithdrawPageTestCases:5]: Expected results should not be stored in Pre-condition section. Normally this section either says something like: 
- User is on Some Page
- User is logged as Admin
Or you can list actions here like: 
- Login as admin 
- Navigate to some page
- Create Transaction

File: src/test/resources/WithdrawPageTestCases [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Step definition:
+- Click "Withdraw" in the left menu bar.
```
Comment (iiddd) [inline src/test/resources/WithdrawPageTestCases:7]: To make your instructions even more clear it would be nice to specify element types (e.g. Click "Withdraw" button from Menu bar)

File: src/test/resources/WithdrawPageTestCases [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- Click "New Withdraw" in the drop-down menu.
+- Check "Withdraw amount" field is empty.
+- Don't select option in the "Puppy" field.
+- Click "Confirm" button.
+Expexted result:
+- We stayed on the "Withdraw Transaction" page.
```
Comment (iiddd) [inline src/test/resources/WithdrawPageTestCases:29]: Is there an error displayed in this case?

File: src/test/resources/WithdrawPageTestCases [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+Step definition:
+- Click "Withdraw" in the left menu bar.
+- Click "New Withdraw" in the drop-down menu.
+- Select "1" in the "Withdraw amount" field.
+- Don't select option in the "Puppy" field.
+- Click "Confirm" button.
+Expexted result:
+- We stayed on the "Withdraw Transaction" page.
+
+5. Withdraw transaction with field "Withdraw amount" is "0"
```
Comment (iiddd) [inline src/test/resources/WithdrawPageTestCases:59]: Looking back to our discuussion about Boundary values analyzis technique, It would be wise to also add a test to withdraw more credits than available I guess. Just a suggestion.

File: src/test/resources/WithdrawPageTestCases (hunk 3 above)
Comment (Manuyafox) [reply src/test/resources/WithdrawPageTestCases:29]: Yes, but the error only appears in the browser. It is not in the code.

**Review by iiddd — APPROVED** [review]
